src/rest-server/src/config/launcher.js

```diff
@@ -194,6 +196,7 @@ if (launcherType === 'yarn') {
   launcherConfig = {
     apiServerUri: apiserver.uri,
     apiVersion: 'frameworkcontroller.microsoft.com/v1',
+    restServerUri: process.env.REST_SERVER_URI,
```

Comment:
it's not a launcher config, put it in `src/config/index.js`

Reply:
Sure

Reply:
pls delete here

Reply:
Fixed

---
In reply to: [349982348](https://github.com/microsoft/pai/pull/3875#discussion_r349982348) [](ancestors = 349982348)